lib/cocina_display/contributors/role.rb

```diff
@@ -5,18 +5,28 @@ module Contributors
     # A role associated with a contributor.
     class Role
       attr_reader :cocina
+      DEFAULT_ROLE = "associated with"
 
       # Initialize a Role object with Cocina structured data.
       # @param cocina [Hash] The Cocina structured data for the role.
       def initialize(cocina)
         @cocina = cocina
       end
 
+      def self.create_roles(roles)
+        Array(roles.presence || [{"value" => DEFAULT_ROLE}]).map { |role| new(role) }
+      end
+
       # The name of the role.
       # Translates the MARC relator code if no value was present.
-      # @return [String, nil]
+      # Otherwise returns the default role.
+      # @return [String]
       def to_s
-        cocina["value"] || (Vocabularies::MARC_RELATOR[code] if marc_relator?)
+        @to_s ||= cocina["value"] || (Vocabularies::MARC_RELATOR[code] if marc_relator?) || DEFAULT_ROLE
+      end
```

Comment:
Is memoization necessary here? Feels kind of weird for something as simple as `#to_s`.

Reply:
Well, it is potentially going through the marc relator logic.  Since we're iterating through all of the Roles (in `Contributor`) to call `#blank?`, which calls`#to_s`, it seems like a good idea.